prespective_rectification/main.py

```diff
@@ -0,0 +1,58 @@
+import cv2
+import matplotlib.pyplot as plt
+import numpy as np
+
+from absl import app, flags
+from absl.flags import FLAGS
+
+from prespective_rectification.abrupt_corners import abrupt_changes_algorithm
+from prespective_rectification.hough_transform_intersections import hough_lines_main
+from prespective_rectification.ploting import plot_points
+from prespective_rectification.pre_process import pre_process
+from prespective_rectification.utils import sort_points, filter_intersections, get_intersection
+
+flags.DEFINE_string('algorithm', 'hough_lines', 'hough_lines or abrupt_changes')
+flags.DEFINE_string('image', 'dataset/plate.png', 'path to input image')
+
+
+def main(_argv):
+    """
+    Run the algorithm of automatic rectification. Steps:
+    - Read and preprocess the image until we have a binaryzed central square object
+    - Run one of the algorithm to extract 4 lines corresponding to each of the corners
```

Comment:
Do the 4 lines correspond to the corner or to the straight lines on the contour of the license plates?